Added little explanation in description (#29799)
ytdl-org/youtube-dl: CONTRIBUTING.md

                # TODO more properties (see youtube_dl/extractor/common.py)
            }
    ```

> 5. Add an import in [`youtube_dl/extractor/extractors.py`](https://github.com/ytdl-org/youtube-dl/blob/master/youtube_dl/extractor/extractors.py). This makes the extractor available for use, on the condition the class ends with IE.

Reviewer: ```suggestion
5. Add an import in [`youtube_dl/extractor/extractors.py`](https://github.com/ytdl-org/youtube-dl/blob/master/youtube_dl/extractor/extractors.py). This makes the extractor available for use, as long as the class ends with `IE`.
```
